experimental/dds/tree2/src/codec/codec.ts

```diff
@@ -65,12 +65,13 @@ export interface JsonValidator {
  */
 export interface ICodecOptions {
 	/**
-	 * JSON Validator which should be used to validated persisted data SharedTree handles.
+	 * {@link JsonValidator} which should be used to validate persisted data SharedTree reads & writes
+	 * matches the expected encoded format (i.e. the wire format for ops and summaries).
```

Comment:
Maybe this can be phrased avoiding "should" (who should?) - maybe something like
```suggestion
	 * When a {@link JsonValidator} is provided, SharedTree will use it internally to validate the persisted data it reads & writes.  If [whatever reasonable condition] then you may choose to omit it, causing SharedTree to skip this internal validation step.
```